### ISSUE TITLE
default_minter probably shouldn't set minter

### ISSUE THREAD

**Issue by atz** [projecthydra-labs/active_fedora-noid#20]
See: https://github.com/projecthydra-labs/active_fedora-noid/blob/master/lib/active_fedora/noid/service.rb#L22-L24

```ruby
      def default_minter
        @minter ||= ActiveFedora::Noid::SynchronizedMinter.new
      end
```

This means the instance variable is set twice during initialization and that the `default_minter` method has an unnecessary side-effect.

**Comment by mjgiarlo** [comment]
Yep, that's a mistake. It should not know about the ivar.

**Comment by atz** [comment]
Addressed in #25
